doc/src/manual/ranch.get_transport_options.asciidoc

```diff
@@ -9,7 +9,7 @@ ranch:get_transport_options - Get the current transport options
 [source,erlang]
 ----
 get_transport_options(Ref :: ranch:ref())
-    -> TransOpts :: any()
+    -> TransOpts :: transport_options(any())
```

Comment:
That's not a real type. You probably refer to https://github.com/ninenines/ranch/blob/master/doc/src/manual/ranch.asciidoc#transport_optssocketopts

Reply:
Right. It should be `ranch:transport_opts(any())`, since the socket opts is any(). I'll update.